src/Hook.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\YouAreUsingItWrong;
+
+use Composer\Composer;
+use Composer\EventDispatcher\EventSubscriberInterface;
+use Composer\IO\IOInterface;
+use Composer\Plugin\PluginInterface;
+use Composer\Script\Event;
+use Composer\Script\ScriptEvents;
+use PackageVersions\Versions;
+use Psalm\Internal\Analyzer\ProjectAnalyzer;
+use Psalm\Internal\Provider\FileProvider;
+use Psalm\Internal\Provider\Providers;
+use Psalm\IssueBuffer;
+use Roave\YouAreUsingItWrong\Composer\Project;
+use Roave\YouAreUsingItWrong\Psalm\Configuration;
+use Roave\YouAreUsingItWrong\Psalm\ProjectFilesToBeTypeChecked;
+use RuntimeException;
+use function define;
+use function defined;
+use function getcwd;
+use function microtime;
+
+/** @internal this is a composer plugin: do not rely on it in your sources */
+final class Hook implements PluginInterface, EventSubscriberInterface
+{
+    private const THIS_PACKAGE_NAME = 'roave/you-are-using-it-wrong';
+
+    /**
+     * {@inheritDoc}
```

Comment:
There are inconsistenciesin the `@inheritDoc` notations. Some are kept on single lines, others are multi line.

Personally i dont'see the point in them, and i'd say delete them all toghether.

Reply:
They are required by `doctrine/coding-standard`, since it isn't clear to the tool if we have control over the signature or not, so `{@inheritDoc}` is to be uaed whenever you don't soecify the API in your subtype.

Besides that, will fix the inconsistencies